.golangci.yml

```diff
@@ -141,7 +141,6 @@ linters:
     - unparam
     - unused
     - varnamelen
-    - wsl
```

Comment:
please remove this change to the lint config too

Reply:
Updated, Thanks!